tests/functional/aws-node-sdk/lib/utility/website-util.js

```diff
@@ -1,6 +1,206 @@
+import assert from 'assert';
 import async from 'async';
 import fs from 'fs';
 import path from 'path';
+import url from 'url';
+
+import makeRequest from '../../../raw-node/utils/makeRequest';
+
+let awsCredentials;
+
+function _parseConfigValue(string, fileSlice) {
+    const line = fileSlice.find(element => element.indexOf(string) > -1);
+    const keyValue = line.replace(/ /g, '').split('=');
+    // the element after the '=' should be the value
+    return keyValue[1];
+}
+
+function _retrieveAWSCredentials(profile) {
+    const filename = path.join(process.env.HOME, '/.aws/scality');
+    let file;
+
+    try {
+        file = fs.readFileSync(filename, 'utf8');
+    } catch (e) {
+        const msg = `AWS credential file does not exist: ${filename}`;
+        throw new Error(msg);
+    }
+
+    const fileContents = file.split('\n');
+    const profileIndex = file.indexOf(`[${profile}]`);
+    if (profileIndex > -1) {
+        const accessKey = _parseConfigValue('aws_access_key_id',
+            fileContents.slice(profileIndex));
+        const secretKey = _parseConfigValue('aws_secret_access_key',
+            fileContents.slice(profileIndex));
+        return { accessKey, secretKey };
+    }
+    const msg = `Profile ${profile} does not exist in AWS credential file`;
+    throw new Error(msg);
+}
+
+if (process.env.AWS_ON_AIR) {
+    awsCredentials = _retrieveAWSCredentials('default');
+}
+
+function _makeWebsiteRequest(auth, method, urlstring, callback) {
+    let authCredentials;
+    if (auth === 'validAuth') {
+        authCredentials = awsCredentials || {
+            accessKey: 'accessKey1',
+            secretKey: 'verySecretKey1',
+        };
+    } else if (auth === 'invalidAuth') {
+        authCredentials = {
+            accessKey: 'fakeKey1',
+            secretKey: 'fakeSecretKey1',
+        };
+    } else if (!auth) {
+        authCredentials = undefined;
+    } else {
+        throw new Error(`Unsupported auth type ${auth}`);
+    }
+    const { hostname, port, path } = url.parse(urlstring);
+    makeRequest({ hostname, port, method, path, authCredentials }, callback);
+}
+
+function _assertResponseHtml(response, elemtag, content) {
+    if (elemtag === 'ul') {
+        const startIndex = response.indexOf('<ul>');
+        const endIndex = response.indexOf('</ul>');
+        assert(startIndex > -1 && endIndex > -1, 'Did not find ul element');
+        const ulElem = response.slice(startIndex + 4, endIndex);
```

Comment:
_Suggestion_: 
```
const startingTag = '<ul>'
const startingTagLength = startingTag.length
```
So, we understand where this "4" comes from and also if we change the tag we won't need to change its length.